RecoTracker/FinalTrackSelectors/plugins/TrackListMerger.cc

```diff
@@ -361,6 +361,8 @@ void TrackListMerger::produce(edm::Event& e, const edm::EventSetup& es) {
     return;
   }
 
+  statCount.begin(rSize);
```

Comment:
What is the reason to move this?

Reply:
to put it back where it was.

Reply:
Ah, because you are reverting https://github.com/cms-sw/cmssw/pull/39233
Ok, got it.